frontend/src/components/navbar/Navbar.tsx

```diff
@@ -15,17 +22,34 @@ export const Navbar: React.FunctionComponent = () => {
           alt="Whale Spotting logo"
         />
       </Link>
-      <div>
+      <span className={`menu-items ${menuCollapsed && "menu-collapsed"}`}>
+        <Link className="menu-link" to="/report-sighting">
+          Report Sighting
+        </Link>
+        <Link className="menu-link" to="/ticket-booking">
+          Book Tickets
+        </Link>
+        <Link className="menu-link" to="/sightings-list">
+          Sightings List
+        </Link>
         {!loginContext.isLoggedIn ? (
-          <div></div>
-        ) : (
-          <Link to="/">
-            <a className="button is-primary" onClick={loginContext.logOut}>
-              <strong>Logout</strong>
-            </a>
+          <Link className="menu-link" to="/login">
+            Login
           </Link>
+        ) : (
+          <>
+            <Link className="menu-link" to="/pending-sights">
+              Pending Sightings
+            </Link>
+            <Link onClick={loginContext.logOut} className="menu-link" to="/">
```

Comment:
Should this be a `<Link>`? Do we actually want to redirect the user to a new page when they log out?